src/sidebar/app/components/Footer.js

```diff
@@ -0,0 +1,333 @@
+import React from 'react';
+
+import {getPadStats} from '../editor.js';
+
+/**
+ * Formats time for the Notes footer
+ * @param time
+ * @returns {string}
+ */
+function formatFooterTime(date) {
+  date = date || Date.now();
+  return new Date(date).toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'});
+}
+
+/**
+ * Set animation on footerButtons toolbar
+ *
+ *
+ * @param {Boolean} animateSyncIcon Start looping animation on sync icon
+ * @param {Boolean} syncingLayout   if true, animate to syncingLayout (sync icon on right)
+ *                                  if false, animate to savingLayout (sync icon on left)
+ * @param {Boolean} warning         Apply yellow warning styling on toolbar
+ */
+function setAnimation( animateSyncIcon = true, syncingLayout, warning ) { // animateSyncIcon, syncingLayout, warning
```

Comment:
nit: can we please remove the extra spaces between args here, as above there are no spaces in `formatFooterTime(date)` between `(` `)` and `date`

Reply:
Prettier should be used to make sure this is no longer a problem. I guess some configurations are needed. So you think travis should fail pull request if prettier return syntax error ?